iotdb-core/datanode/src/test/java/org/apache/iotdb/db/storageengine/dataregion/compaction/CompactionWorkerTest.java

```diff
@@ -261,4 +266,30 @@ public void testFailedToCheckValidInInnerTask() throws InterruptedException {
     thread.interrupt();
     thread.join();
   }
+
+  @Test
+  public void testAbortCompactionTask() {
+    AtomicReference<CompactionWorker.CompactionTaskFuture> compactionTaskSummary =
+        new AtomicReference<>();
+    AtomicBoolean isInterrupted = new AtomicBoolean(false);
+    Phaser phaser = new Phaser(2);
+    Thread t =
+        new Thread(
+            () -> {
+              compactionTaskSummary.set(
+                  new CompactionWorker.CompactionTaskFuture(new CompactionTaskSummary()));
+              phaser.arriveAndAwaitAdvance();
+              try {
+                Thread.sleep(10000);
```

Comment:
This can be unstable. If the test is hung up for 10s or there is a clock problem in the test machine, this test is likely to fail.